src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -69,13 +75,15 @@ class IpexCommunicationService extends AgentService {
     const exn = await this.props.signifyClient
       .exchanges()
       .get(notifRecord.a.d as string);
+
     const credentialId = exn.exn.e.acdc.d;
     const connectionId = exn.exn.i;
     await this.saveAcdcMetadataRecord(
       exn.exn.e.acdc.d,
       exn.exn.e.acdc.a.dt,
       connectionId
     );
+    await this.createLinkedIpexMessageRecord(exn.exn.i, exn);
```

Comment:
AC: "The connection history interface should show presentation requests, presentations from the other party, issuances from the other party" - so apply messages too. 

But I think we can leave the presentations. What happens right now if we receive a grant message into our wallet that is a presentation and not an issuance?